lib/token-request.js

```diff
@@ -559,7 +566,10 @@ TokenRequest.prototype.getTokenWithCertificate = function(certificate, thumbprin
   oauthParameters[OAuth2Parameters.CLIENT_ASSERTION_TYPE] = OAuth2GrantType.JWT_BEARER;
   oauthParameters[OAuth2Parameters.CLIENT_ASSERTION] = jwt;
 
-  this._getTokenWithCacheWrapper(callback, function(getTokenCompleteCallback) {
+  this._getTokenWithCacheWrapper(callback, function (err, getTokenCompleteCallback) {
+    if (err) {
```

Comment:
okay, i can back out this change. Do you want me to back all except the "getTokenFromCacheWithRefresh"? Which is the main case i care the most about.
I did a little more for the main reason of being consistent and related readibility. Backing out is very fine by me